### ISSUE TITLE
T2: Multi-asic support needed for telemetry/test_events , test_telemetry scripts

### ISSUE THREAD

**Issue by rajendrat** [sonic-net/sonic-mgmt#12083]
**Description**

telemetry/test_events , test_telemetry scripts are failing in multi-asic switches.
coniguration and show commands used in the test does not use namespace option associated with the asic.

**Steps to reproduce the issue:**
1. Run the telemetry/test_events , test_telemetry  tests
2.
3.

**Describe the results you received:**

Error:

test_events error:
        if (res.is_failed or 'exception' in res) and not module_ignore_errors:
&gt;           raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), res)
E           tests.common.errors.RunAnsibleModuleFail: run module shell failed, Ansible Results =&gt;
E           {"changed": true, "cmd": "config interface startup Ethernet0", "delta": "0:00:01.210832", "end": "2024-03-19 1
3:12:20.301258", "failed": true, "msg": "non-zero return code", "rc": 2, "start": "2024-03-19 13:12:19.090426", "stderr": 
"Usage: config interface [OPTIONS] COMMAND [ARGS]...\nTry \"config interface -h\" for help.\n\nError: Missing option \"-n\
" / \"--namespace\".  Choose from:\n\tasic0,\n\tasic1,\n\tasic2.", "stderr_lines": ["Usage: config interface [OPTIONS] COM
MAND [ARGS]...", "Try \"config interface -h\" for help.", "", "Error: Missing option \"-n\" / \"--namespace\".  Choose fro
m:", "\tasic0,", "\tasic1,", "\tasic2."], "stdout": "", "stdout_lines": []}

    
test_telemetry error:

        if (res.is_failed or 'exception' in res) and not module_ignore_errors:
&gt;           raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), res)
E           tests.common.errors.RunAnsibleModuleFail: run module command failed, Ansible Results =&gt;
E           {"changed": true, "cmd": ["vtysh", "-c", "show bgp ipv6 neighbor fc00::72 json"], "delta": "0:00:00.006071", "
end": "2024-03-19 13:24:43.564359", "failed": true, "msg": "non-zero return code", "rc": 1, "start": "2024-03-19 13:24:43.
558288", "stderr": "Usage: /usr/bin/vtysh -n [0 to 2] [OPTION]... ", "stderr_lines": ["Usage: /usr/bin/vtysh -n [0 to 2] [
OPTION]... "], "stdout": "", "stdout_lines": []}

**Describe the results you expected:**
The script check if its a multi-asic or not used appropriate namespace.

**Comment by rajendrat** [comment]
viz @abdosi @vperumal

**Comment by abdosi** [comment]
need to enhance for multi-asic.

**Comment by zbud-msft** [comment]
ETA: 5/26

**Comment by rlhui** [comment]
This PR is to fix this https://github.com/sonic-net/sonic-mgmt/pull/13826

**Comment by rlhui** [comment]
Can https://github.com/sonic-net/sonic-mgmt/pull/14982 address this?

**Comment by rlhui** [comment]
@wumiaont would you please help confirm if this is fixed now? thanks.

**Comment by wumiaont** [comment]
> [@wumiaont](https://github.com/wumiaont) would you please help confirm if this is fixed now? thanks.

Yes. test_telemetry.py related multi-asic issues has been fixed by 14982. test_events most of tests are skipped for multi-asic because of events are not ready for multi-asic support yet. 

We can close this issue.
